apps/web/server/routers/viewer.tsx

```diff
@@ -394,13 +430,23 @@ const loggedInViewerRouter = createProtectedRouter()
           rescheduled: true,
         },
         orderBy,
+        distinct,
         take: take + 1,
         skip,
       });
 
+      const groupedRecurringBookings = await prisma.booking.groupBy({
```

Comment:
Does this query is missing a `where`?

Reply:
Not really, I'm getting all recurringEventIds to count them and return them all to then iterate through them